.travis.yml

```diff
@@ -1,19 +1,21 @@
+sudo: false
 os:
   - linux
-  - osx
```

Comment:
OSX is delaying builds on Travis so I disable it in repos that don't have code that might depend on the system.